resources/email.py

```diff
@@ -32,3 +31,19 @@ def send_reset_password_msg(user):
         msg.html = render_template('emails/reset_msg.html', user=user, token=token)
 
         current_app.mail.send(msg)
+
+        return {"message": "Message sent"}
+
+class InviteEmail(Resource):
+    parser = reqparse.RequestParser()
+    parser.add_argument('email', type=str, required=True, help="This field is not required.")
+
+    def post(self):
+        data = InviteEmail.parser.parse_args()
+
+        message = Message("You've been invited to join Dwellingly!", sender="noreply@codeforpdx.org", html="<a href=\"http://localhost:3000/signup\">Sign up for Dwellingly</a>")
```

Comment:
Let's use a constant instead of the hard-coded string: "http://localhost:3000"  - If we don't have an env var for this then let's create one. 

Also it would probably be good to send both html and text versions of the email, and we probably want to have some kind of message. <- We can always do this in another ticket though. @htharker42 do you have a message that should be used?